underfs/cephfs/pom.xml

```diff
@@ -31,7 +31,6 @@
       <groupId>io.github.opendataio</groupId>
       <artifactId>libcephfs</artifactId>
       <version>0.0.1</version>
-      <scope>provided</scope>
```

Comment:
What's the purpose you remove the `provided ` scope, you want to include the `libcephfs ` and related share lib into the ufs shaded with dependencies jar?

Reply:
> What's the purpose you remove the `provided ` scope, you want to include the `libcephfs ` and related share lib into the ufs shaded with dependencies jar?

Yes, this pr will solve the problem about the libcephfs-jni.so is already loaded in another classloader, so libcephfs can be included.